### PR TITLE
Chore/delta engine installer

### PR DESCRIPTION
**Pull request by rnithinpaladin**
https://paladincloud.atlassian.net/browse/NEXT-43

delta engine installer code 

## Summary by CodeRabbit

- **New Features**
	- Introduced a new service `svc-risk-score` with API configurations.
	- Added a command-line utility for managing AWS CloudFormation stacks.
	- New build and deployment workflows for the asset shipper JAR.
- **Improvements**
	- Updated configurations for existing services to use explicit boolean values.
	- Enhanced deployment process with structured build specifications for AWS CodeBuild.
- **Bug Fixes**
	- Corrected environmental configurations for services to ensure proper functionality.

### CODE REVIEW

**Comment by rnithinpaladin** [comment]
> @rnithinpaladin - you should automate these steps to make it reusable to any service to be installed. More, you should be installing a subsystem and not an individual service. Approved to unblock the delivery and deployments, however, we should discuss and review the requirements.

yes... I'm working on generic installer for all subsystem 
1.  Workflow design to build jar file or deploy layers 
2.  Individual pipeline for subsystem  
3.  generating cloudformation stack

File: installer/delta-engine/delta-engine.yaml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  LambdaLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties: 
+      LogGroupName: !Sub "/aws/lambda/svc-${Environment}-asset-shipper"
+      RetentionInDays: 14
+
+  SQSAssetShippingDone:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub "svc-${Environment}-asset-shipping-done.fifo"  # Use !Sub for environment substitution
```
Comment (plyubich) [inline installer/delta-engine/delta-engine.yaml:132]: At another revision, we should feed the queue name from the service configuration in config.yaml. It's standard queue name should be ${Environment}-{value_from_config}.

for this revision, you can correct it to ${Environment}-svc-asset-delta-engine-done.fifo

File: installer/delta-engine/delta-engine.yaml [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+      FifoQueue: true
+      ContentBasedDeduplication: true      
+      SqsManagedSseEnabled: true
+      VisibilityTimeout: 900
+
+  SQSAssetMappingDone:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub "svc-${Environment}-asset-mapping-done.fifo"  # Use !Sub for environment substitution
```
Comment (plyubich) [inline installer/delta-engine/delta-engine.yaml:141]: see comment above

File: installer/delta-engine/delta-engine.yaml [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    Description: The ARN of the Lambda function.
+    Value: !GetAtt svcassetshipper.Arn
+    Export:
+      Name: LambdaFunctionArn
+
+  AssetShipperSQSQueueURL:
+    Description: The URL of the svc-asset-shipping-done.fifo SQS queue.
+    Value: !Ref SQSAssetShippingDone
+    Export:
+      Name: AssetShipperSQSQueueURL
```
Comment (plyubich) [inline installer/delta-engine/delta-engine.yaml:177]: need to standardize all variable to reflect the service name, or queue name. 

e.g., AssetDeltaEngineDoneSqs or similar, just needs to follow the same convension.

**Review by plyubich — DISMISSED** [review]
so minor changes and thoughts that you can take in the next revision. As long as this builds and deploys to all environments successfully today, its good to go out.

**Pull request merged**